File: fibonacci/fibonacci.py
import sys

def fibonacci(n):
    a, b = 0, 1
    for _ in range(n):
        a, b = b, a + b
    return a

if __name__ == "__main__":
    # Increase the limit for integer string conversion
    sys.set_int_max_str_digits(0)  # 0 means no limit
    result = fibonacci(100000)
    print(result)

File: fibonacci/fibonacci.go
package main

import "fmt"

func fibonacci(n int) uint64 {
	a, b := uint64(0), uint64(1)
	for i := 0; i < n; i++ {
		a, b = b, a+b
	}
	return a
}

func main() {
	result := fibonacci(100000)
	fmt.Println(result)
}
